gulpfile.js

```diff
@@ -37,9 +38,9 @@ gulp.task('fetch', async () => {
   });
   let fileContents = `// Last snapshot: ${new Date()}\n`;
   fileContents += `const OPTIMIZELY_DATA = ${JSON.stringify(response)};`;
```

Comment:
See above, but I dont think we should include the data at build time, but rather use the SDK key to download it at runtime.